Sources/Client/Model/Channel/Channel+Requests.swift

```diff
@@ -284,6 +284,37 @@ public extension Channel {
         Client.shared.remove(members: members, from: self, completion)
     }
     
+    /// Query this channel's members.
+    /// - Parameters:
+    ///   - filter: Filter conditions for query
+    ///   - sorting: Sorting conditions for query
+    ///   - limit: Limit for number of members to return. Defaults to 100.
+    ///   - offset: Offset of pagination. Defaults to 0.
+    ///   - completion: Completion block with `MembersQueryResponse`
+    @discardableResult
+    func queryMembers(filter: Filter,
+                      sorting: [Sorting] = [],
+                      limit: Int = 100,
+                      offset: Int = 0,
+                      _ completion: @escaping Client.Completion<MembersQueryResponse>) -> Cancellable {
+        let query: MembersQuery
+        if !id.isEmpty {
```

Comment:
nit: why do you use `if !isEmpty { } else { }` when you can use `if isEmpty { } esle { }`? :)